pkg/patterns/addon/pkg/loaders/fs.go

```diff
@@ -46,12 +46,12 @@ func NewManifestLoader() *ManifestLoader {
 	return &ManifestLoader{repo: repo}
 }
 
-func (c *ManifestLoader) ResolveManifest(ctx context.Context, object runtime.Object) (string, error) {
+func (c *ManifestLoader) ResolveManifest(ctx context.Context, object runtime.Object) (map[string]string, error) {
```

Comment:
Could you add test coverage for this (and/or LoadManifest) now that it's a bit more complex?

Reply:
Yes. I will add it in next commit.